apps/studio/src/components/PageEditor/ComponentSelector.tsx

```diff
@@ -92,29 +93,32 @@ const BlockItem = ({
   )
 }
 
-interface ComponentSelectorProps {
-  onClose: () => void
-  onProceed: (sectionType: SectionType) => void
-}
-
-const ComponentSelector = ({ onClose, onProceed }: ComponentSelectorProps) => {
+function ComponentSelector() {
+  const { setDrawerState, setPageState, setEditorState } =
+    useEditorDrawerContext()
+  const onProceed = (sectionType: SectionType) => {
+    // setPageState()
+    // setEditorState()
+    console.log(sectionType)
+    setDrawerState({ state: 'root' })
```

Comment:
Hmm this is meant to do 2 things - to add the new block to the page, then return to the root state. Since we don't have the page state properly setup yet it only goes back to the root drawer state